otx/core/data/adapter/detection_dataset_adapter.py

```diff
@@ -50,7 +50,7 @@ def get_otx_dataset(self) -> DatasetEntity:
                         if ann.label not in used_labels:
                             used_labels.append(ann.label)
 
-                    if len(shapes) > 0 or subset == Subset.UNLABELED or len(datumaro_item.annotations) == 0:
+                    if len(shapes) > 0 or subset == Subset.UNLABELED or (subset == Subset.VALIDATION and len(datumaro_item.annotations)) == 0:
```

Comment:
```suggestion
                    if len(shapes) > 0 or subset == Subset.UNLABELED or (subset == Subset.VALIDATION and len(datumaro_item.annotations) == 0):
```

Reply:
how about checking it's not training dataset not to filter out image w/o annotation in test dataset.

Reply:
I can't understand what you want to say.. could you explain more detail?

Reply:
So current change just keep images w/o annotation only in validation dataset but not in test dataset. To keep them in both case, how about checking `subset != Subset.TRAINING` instead of `subset == Subset.VALIDATION`?